index.js

```diff
@@ -105,12 +90,16 @@ app.use(
 app.use(
   session({
     secret: process.env.SESSION_KEY || "meowmeow",
+    resave: true,
+    saveUninitialized: true
```

Comment:
```suggestion
    resave: false,
    saveUninitialized: false
```

Avoids [saving cookies when not needed](https://github.com/expressjs/session#resave)